src/main/java/businessLayer/userTypes/SystemController.java

```diff
@@ -151,7 +151,8 @@ public boolean sendAlert(List<Subscriber> followers, List<String> Alerts) {
     public Boolean insertInfo(String userName, String password) {
         if (userName.equals("admin") && password.equals("admin")) {
             temporaryAdmin = new Admin(userName, password, "tempAdmin", this);
-            temporaryAdmin.setApproved(true);
+            ((Admin) temporaryAdmin).setApproved(true);
+            DB.addSubscriberToDB("admin",temporaryAdmin);
```

Comment:
maybe we should provide here a better explanation about this function, that it is being used only when we start the system in the first time